src/main/java/appeng/parts/p2p/PartP2PTunnelME.java

```diff
@@ -232,7 +232,9 @@ else if( !cw.getTunnel().getProxy().isActive() )
 					{
 						final TileEntity start = this.getTile();
 						final TileEntity end = me.getTile();
-						AELog.warn( "Failed to establish a ME P2P Tunnel between the tunnels at [x=%d, y=%d, z=%d] and [x=%d, y=%d, z=%d]", start.xCoord, start.yCoord, start.zCoord, end.xCoord, end.yCoord, end.zCoord );
+						AELog.warn( "Failed to establish a ME P2P Tunnel between the tunnels at [x=%d, y=%d, z=%d, dim=%d] and [x=%d, y=%d, z=%d, dim=%d]",
+								start.xCoord, start.yCoord, start.zCoord, start.hasWorldObj() ? start.getWorldObj().provider.dimensionId : Integer.MAX_VALUE,
```

Comment:
How could world obj be null though?

Reply:
not really sure if it can (it definitely cannot for the source points), but for the end points retrieving them is hidden somewhere far away that I was too lazy to dig